### PR TITLE
Write blog post about Strimzi at KubeCon in San Diego

### PR DESCRIPTION
**Pull request by scholzj**
This PR adds the blog post about Strimzi presence at KubeCOn in San Diego.

### CODE REVIEW

File: _posts/2019-11-17-strimzi-at-kubecon-san-diego.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* On **Thursday at 11:10am**
+
+_(The full demo schedule is available [here](https://www.redhat.com/en/events/red-hat-kubecon-cloudnativecon-north-america-2019#demo-schedule))._
+
+After the demo, we will be available for any questions or discussions related to Strimzi and Apache Kafka.
+
+Of course, we know that KubeCon will be full of interesting talks.
+So if you cannot make it to any of the demos because of some other talk, don't worry.
+Jakub Scholz - one of our maintainers - will be available through the whole conference.
+So if you want to meet and talk about Strimzi or Aapche Kafka, just ping him on the #strimzi channel on [CNCF Slack](https://slack.cncf.io/) or at Twitter ([@strimziio](https://twitter.com/strimziio) or [@scholzj](https://twitter.com/scholzj).
```
Comment (ppatierno) [inline _posts/2019-11-17-strimzi-at-kubecon-san-diego.md]: typo "Aapche"

File: _posts/2019-11-17-strimzi-at-kubecon-san-diego.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Strimzi at KubeCon+CloudNativeCon in San Diego"
+date: 2019-11-17
```
[the remark this answers is not in the document]
Comment (scholzj) [reply _posts/2019-11-17-strimzi-at-kubecon-san-diego.md]: I can change that later, but I want to see the preview.

File: _posts/2019-11-17-strimzi-at-kubecon-san-diego.md (hunk 1 above)
Comment (scholzj) [reply _posts/2019-11-17-strimzi-at-kubecon-san-diego.md]: You are too slow as a reviewer ... fixed looooong time ago ;-)

File: _posts/2019-11-17-strimzi-at-kubecon-san-diego.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+author: strimzi_developers
+---
+
+From Tuesday, most of the Cloud Native and Kubernetes world meets in San Diego for the [KubeCon+CloudNativeCon conference](https://events19.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/) organized by Cloud Native Computing Foundation (CNCF). 
+As a CNCF Sandbox project, Strimzi will be of course represented there as well.
```
Comment (tombentley) [inline _posts/2019-11-17-strimzi-at-kubecon-san-diego.md]:
```suggestion
As a CNCF Sandbox project, Strimzi will, of course, be represented there as well.
```

File: _posts/2019-11-17-strimzi-at-kubecon-san-diego.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Read more to learn how to meet Strimzi maintainers and see Strimzi demos.
+
+<!--more-->
+
+If you are attending KubeCon, you have two opportunities to see live demo of Strimzi in the mini-theatre at [Red Hat booth (#D1)](https://events19.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/venue-map/):
```
Comment (tombentley) [inline _posts/2019-11-17-strimzi-at-kubecon-san-diego.md]:
```suggestion
If you are attending KubeCon, you have two opportunities to see a live demo of Strimzi in the mini-theatre at the [Red Hat booth (#D1)](https://events19.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/venue-map/):
```

**Pull request merged**